packages/bpk-component-autosuggest/package.json

```diff
@@ -18,7 +18,7 @@
     "bpk-mixins": "^20.1.17",
     "bpk-react-utils": "^3.0.11",
     "prop-types": "^15.6.2",
-    "react-autosuggest": "^10.0.2"
+    "react-autosuggest": "^9.4.3"
```

Comment:
What's to prevent this being upgraded again in the future?
Could we write a test that verifies the version. The test can also serve as documentation with a comment to say why we're testing it 👍

Reply:
That is a good shout will add that now!

Reply:
You can use `import packageJson from './package.json';` and then just read the value, and assert that it's `"^9.4.3"`